bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/resources/PdfUploader.kt

```diff
@@ -0,0 +1,35 @@
+package net.gini.android.bank.sdk.exampleapp.ui.resources
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiSelector
+
+class PdfUploader {
+    private val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+    fun uploadPdfFromFiles() {
+        //Wait for 'tab'
+        device.waitForIdle()
+        val tabTitle = device.findObject(UiSelector().className("android.widget.TextView").text("Recent"))
+        if (tabTitle.exists() && tabTitle.text == "Recents") {
```

Comment:
The `tabTitle.exists() ` would always return `true` because each and every tab is part of same class `android.widget.TextView`.
The `tabTitle.text == "Recents"` would help to make obvious to execute the `if` clause.

Reply:
I agree, but `findObject(UiSelector().className("android.widget.TextView").text("Recent"))` also contains a check for `Recent` to be the text of the tab title. 

Would this not return false if there was no tab with `Recent` text on it?